src/wp-admin/includes/class-wp-comments-list-table.php

```diff
@@ -663,7 +664,7 @@ public function single_row( $item ) {
 	 *                or if the current user cannot edit the comment.
 	 */
 	protected function handle_row_actions( $item, $column_name, $primary ) {
-		global $comment_status;
+		global $comment_status, $hook_suffix;
```

Comment:
```suggestion
		global $comment_status, $hook_suffix;
		
		$hook_suffix = 'comments-row-actions';
```
This should fix the failing PHP 8.1 tests.

@peterwilsoncc Looks like these failures appeared because PHP 8.1 was removed from `continue-on-error` in [54072](https://core.trac.wordpress.org/changeset/54072).

This exposes [this line](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/includes/class-wp-screen.php#L223) once again.

From what I can tell, proper checking here would be:
```php
		if ( $hook_name ) {
			$id = $hook_name;
		} elseif ( wp_doing_ajax() && array_key_exists( 'action', $_REQUEST ) ) {
			$id = 'wp_ajax_' . esc_attr( $_REQUEST['action'] );
		} elseif ( array_key_exists( 'hook_suffix', $GLOBALS ) ) {
			$id = $GLOBALS['hook_suffix'];
		} else {
			$id = '';
		}
```

We should really update `WP_Screen::get()` with appropriate checks given this and the above tickets.